mhcflurry/class1_affinity_prediction/class1_affinity_predictor.py

```diff
@@ -119,6 +119,27 @@ def supported_peptide_lengths(self):
             max(lower for (lower, upper) in length_ranges),
             min(upper for (lower, upper) in length_ranges))
 
+    def merge(self, affinity_predictors=[]):
+        """ 
+        Merge multiple affinity predictors into one.  
+        When training MHCFlurry in parallel, the user might be left with
+        multiple affinity predictors, merge can take in multiple affinity predictors
+        and the result will be a single ensemble.
+        Parameters
+        ----------
+        affinity_predictors : array
+            Affinity predictors that want to merge
+        """
+        self.allele_to_allele_specific_models
+        for affinity_predictor in affinity_predictors:
+            for allele in affinity_predictor.allele_to_allele_specific_models.keys():
+                if allele in self.allele_to_allele_specific_models:
```

Comment:
We try to keep lines to not much more than 80 chars (not sure exactly what the rest of the code is doing but I would say try to fit in with the rest of it)